CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 
 All notable changes to the "azurefunctions" extension will be documented in this file.
 
+## 0.9.0 - 2018-05-09
+
+### Added
+
+- Improved warning when Azure Functions Core Tools is out of date
+  - In some cases, an 'Update' option is provided that will automatically run the necessary commmands
```

Comment:
Should we be more specific about the cases?

Reply:
I feel like the extra clutter isn't worth it